environment.yml

```diff
@@ -12,6 +12,7 @@ dependencies:
     - netCDF4
     - packaging
     # Optional dependencies
+    - contextily
```

Comment:
Off-topic: we probably need to move ipython to "Optional dependencies" and remove jupyter from "Development dependencies".

Reply:
Handled in #2293.